internal/signerverifier/ssh/ssh.go

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: Apache-2.0
+package ssh
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"encoding/base64"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/hiddeco/sshsig"
+	"golang.org/x/crypto/ssh"
+)
+
+// FIXME: make this the interface for Keys in TUF metadata
+type Key struct {
+	KeyType string
+	KeyVal  KeyVal
```

Comment:
I would have liked to move `Public` here directly but I think that's more painful for compat than dropping KeyType would be :(

Reply:
If we decide to not be compatible with TUF keys, we can do this too. If we do not want to the door to TUF or TUF key tooling (e.g. securesystemslib), we should maybe not drop `KeyType` either.